test/integration/element.test.js

```diff
@@ -0,0 +1,297 @@
+import {getBrowser} from '../mini-testium-mocha';
+import assert from 'assertive';
+
+function stripColors(message) {
+  return message.replace(/\u001b\[[^m]*m/g, '');
+}
+
+xdescribe('element', () => {
+  let browser;
+  before(async () => (browser = await getBrowser()));
+
+  before(() => browser.navigateTo('/'));
+
+  it('can get an element\'s text', () => {
+    const element = browser.getElement('h1');
+    const text = element.get('text');
+    assert.equal('Element text was not found', 'Test Page!', text);
+  });
+
+  it('can get special properties from an element', () => {
+    // the "checked" property (when it doesn't exist)
+    // returns a non-standard response from selenium;
```

Comment:
Erm, good question. This is copied from `testium-driver-sync` where it was copied from `testium`. I might have more to say about this when I get to porting this file, not sure yet how `wd` and `webdriver-http-sync` handle it. :)